pkg/sql/plan2/build_insert.go

```diff
@@ -14,8 +14,197 @@
 
 package plan2
 
-import "github.com/matrixorigin/matrixone/pkg/sql/parsers/tree"
```

Comment:
How does insert into dest select * from src work?   
I think insert should be just algebraized as,
insert -> insert node, which has a input (that is, child)
select or values -> a query tree

Reply:
Yes, you are right.  I change the code and add the test case in build_test.go in the last commit。thanks。
and yes some corner not cover，I will fixed it in next commit.